flink-libraries/flink-state-processing-api/src/test/java/org/apache/flink/state/api/input/KeyedStateInputFormatTest.java

```diff
@@ -213,7 +213,7 @@ private List<Integer> readInputSplit(
             throws IOException {
         KeyedStateInputFormat<Integer, VoidNamespace, Integer> format =
                 new KeyedStateInputFormat<>(
-                        new OperatorState(OperatorIDGenerator.fromUid("uid"), 1, 4),
+                        new OperatorState(null, null, OperatorIDGenerator.fromUid("uid"), 1, 4),
```

Comment:
Such cases I've considered to add a constructor where `UID` and `name` fields are just hardcoded `null` values (then we wouldn't need so many code changes in tests) but then I've ended up to miss production code parts where the mentioned fields are essential which caused partially working solution. I think having a rock state state story is must have from user's perspective so I've implemented this a more defensive way (compiler blows up not having enough constructor parameters) in order to change all the required execution paths.